### DISCUSSION TITLE
[BUG] file input type does not handle the accept filter correctly

### DISCUSSION THREAD

**Discussion by MarcWeustink** [elgatosf/elgato#81] [Help]
There are several issues with the accept filter:
1. ```<input class="sdpi-item-value" id="select_1" type="file" accept=".jpg">```
Expected: a file choose dialog with a filter: supported formats = *.jpg
Result:  a file choose dialog with no filter

2. ```<input class="sdpi-item-value" id="select_1" type="file" accept=".jpg,.svg">```
Expected: a file choose dialog with a filter: supported formats = *.jpg *.svg
Result: a file choose dialog with a filter: supported formats = *.jpg *.jpeg *.jpe *.svg

3. ```<input class="sdpi-item-value" id="select_1" type="file" accept=".jpg,.jpeg">```
Expected: a file choose dialog with a filter: supported formats = *.jpg *.jpeg
Result: a file choose dialog with a filter: supported formats = *.jpg *.jpeg *.jpe *.jpg *.jpeg *.jpe

4. ```<input class="sdpi-item-value" id="select_1" type="file" accept=".jpg,.foo">```
Expected: a file choose dialog with a filter: supported formats = *.jpg *.foo
Result: a file choose dialog with a filter: supported formats = *.jpg *.jpeg *.jpe

I ran into this since I wanted to add the ability to select our own custom image format.
Thanks, Marc

OS: Win11
SD version: 6.4.1.19697

**Comment by ChekTek** [comment]
Hi @MarcWeustink 

Apologies for the late response! My recommendation would be to try out the new [SDPI components](https://sdpi-components.dev/docs/components/file) file control. Please keep in mind these components are still a work in progress!
